### PR TITLE
Brass Knuckle Melee weapon

### PR DESCRIPTION
**Pull request by VividPups**
## About the PR
Made a new Melee Glove weapon like the Northstars 

## Why / Balance
It was requested, check like the idea and there is only 1 item the Northstars that is like this 

## How to test
spawn it 
## Media
image
image
image
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame

**Changelog**.
:cl:
- add: Brass Knuckles

### CODE REVIEW

**Comment by VividPups** [comment]
um only put it in here because something is broken and its RSI  BS

**Comment by VividPups** [comment]
Yes I know it looks like a hat

**Comment by dvir001** [comment]
Other then the sprite looking a bit too simple (icon) the balance on it is also a bit broken since its using the ``ClothingHandsGlovesNorthStar`` as base.

It means it has contraband worth and the syndicate north star scream on it, this should be rebased on the normal boxer gloves.

**Comment by VividPups** [comment]
> Other then the sprite looking a bit too simple (icon) the balance on it is also a bit broken since its using the `ClothingHandsGlovesNorthStar` as base.
> 
> It means it has contraband worth and the syndicate north star scream on it, this should be rebased on the normal boxer gloves.

the Scream is intend

**Comment by VividPups** [comment]
scream was removed @dvir001

**Comment by Everturning** [comment]
I made the sprites from scratch, btw, if it's anything that you need to change attributions wise

File: Resources/Prototypes/_NF/Recipes/Crafting/brassknuckles.yml
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Crafting/brassknuckles.yml:8]: 1. Why use a different description here?
2. Strangely phrased.  A mouth can't move on its own.

Why even have a construction graph for these?  They aren't some improvised weapons.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:25]: How do brass knuckles mask your fingerprints?  Why do they leave yellow fibers?

**Review by whatston3 — CHANGES_REQUESTED** [review]
Not a fan of these as they exist.  Too strong, weird features, block your fingerprints, no intrinsic value, yet aren't recyclable for materials.

Also, I don't think this sprite looks good.  Here's one I made up with rudimentary shading that I think looks better:
image

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:5]: any type of weapon has a brand on it

File: Resources/Prototypes/_NF/Recipes/Crafting/brassknuckles.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Recipes/Crafting/brassknuckles.yml:8]: because checkraze requested it.  also, yes everyone knows a mouth cant move on it own. it means someone is talking to much shit

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml
Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:5]: Barely any do in-game, save maybe the Argenti, Gestio, and yours.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:17]: it doesnt do 20 damage for second because everything has armor most common have  30 - 60 percent.  not to mention try fighting any of the PVE creatures  like  syndis/mercs/robots.  with brass knuckles most likely you would be near dead or dead

Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:17]: As an apples-to-apples comparison, considering something's base values is fine, and relatively speaking, this is still powerful.

Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:17]: it isnt an apples to apples comparison. in order to get close to the actual damage value I am trying to produce you have to increase the damage

Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:17]: Perhaps melee weapons should be buffed, but standard fare brass knuckles should not be an _unusually strong_ melee weapon.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:25]: Yellow is the closes and if i didnt it would have pull from the parent

Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:25]: Again, how do brass knuckles mask your fingerprints?

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:5]: all basic weapons have the NT brand or the Syndicate fraction Brands.

Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:5]: Save maybe the Argenti, Gestio, and yours :)

Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:5]: Also, why is a craftable item branded?

File: Resources/Prototypes/_NF/Recipes/Crafting/brassknuckles.yml
Comment (Cheackraze) [reply Resources/Prototypes/_NF/Recipes/Crafting/brassknuckles.yml:8]: the description could be better, I agree, but when discussing it on discord I thought it would fit in with the hand-made weaponry like the baseball bat, folding chair, or spear

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml
Comment (Everturning) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:25]: I agree that they shouldn't leave fibers

Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/brassknuckles.yml:25]: how are you wanting me to prevent the pull from the parent

**Comment by whatston3** [comment]
Sam, I've pushed a set of changes up onto `whatston3:brass-knuckles-suggestions` for your review.  I think I'm happy with the state of that branch, but I'd be curious of your thoughts.  [Diff of changes here](https://github.com/VividPups/frontier-station-14/compare/brass-knuckles...whatston3:frontier-station-14:brass-knuckles-suggestions?expand=1), list below.

* Adds improvised brass knuckles, a craftable version, doing 9.6 blunt dps.
* Adds crass knuckles, a craftable joke version, doing 0 blunt dps and 8 stamina dps (it farts on hitting).
* Reduces the dps of brass knuckles down to 12 (between the reinforced and plasma shiv)
* Makes regular and improvised brass knuckles conductors, doing an extra 5% shock damage and 1.5x damage when hacking or punching wires.
* Replaced the icon sprites with my own (more evenly spaced fingers, arguably cleaner shapes).
* Revised the equipped-HAND sprite to use the inhand sprites (which look good!) combined for both hands.
* Adds the brass knuckles to the POI liberation station, and moves both up with the survival knife.
* Comments items added to the regular liberation station inventory.
* Lowercase item names, consistent and shorter descriptions, and a small joke for the crass knuckles description.

image

**Comment by VividPups** [comment]
weapon is ready for second review

**Review by whatston3 — APPROVED** [review]
Marking this as acceptable, seems fine to me, will merge in a day or two if there aren't complaints.

**Comment by VividPups** [comment]
when it been two days and it still isn't merge : (

**Pull request merged**
